### PR TITLE
add subcommand "gopass git remote remove"

### PR DESCRIPTION
**Pull request by campoy**

### CODE REVIEW

**Comment by muesli** [comment]
Generally looks good to me, please see the two comments I left: looks like simple copy/paste mistakes.

The TravisCI build fails because the tests also need be updated when adding a new command. Happy to help you out there, but knowing you I'm almost sure you're already on to it.

Thanks for the PR!

**Comment by dominikschulz** [comment]
@campoy Do think you can address those review comments?

**Comment by campoy** [comment]
Yes, sorry. I sent the PR and went out for the weekend! haha

File: pkg/backend/rcs/git/gogit/git.go [hunk 1]
```diff
@@ -285,6 +285,11 @@ func (g *Git) AddRemote(ctx context.Context, remote, url string) error {
 	return err
 }
 
+// RemoveRemote adds a new remote
```
[the remark this answers is not in the document]
Comment (campoy) [reply pkg/backend/rcs/git/gogit/git.go]: done

File: pkg/backend/rcs/git/cli/git.go [hunk 2]
```diff
@@ -261,6 +261,11 @@ func (g *Git) AddRemote(ctx context.Context, remote, url string) error {
 	return g.Cmd(ctx, "gitAddRemote", "remote", "add", remote, url)
 }
 
+// RemoveRemote adds a new remote
```
[the remark this answers is not in the document]
Comment (campoy) [reply pkg/backend/rcs/git/cli/git.go]: done

**Comment by campoy** [comment]
Ok, this *should* be good now! Sorry for the many mistakes, I ended up having to send the PR in a rush.

**Comment by muesli** [comment]
@campoy It seems you have only merged the latest changes from master into your fork, but I can't seem to find any of the discussed changes?

**Comment by campoy** [comment]
All of the facepalms apply, let's see now.

**Comment by campoy** [comment]
Finally 🎉

File: pkg/action/git.go [hunk 3]
```diff
@@ -88,6 +88,18 @@ func (s *Action) GitAddRemote(ctx context.Context, c *cli.Context) error {
 	return s.Store.GitAddRemote(ctx, store, remote, url)
 }
 
+// GitRemoveRemote removes a new git remote
```
Comment (muesli) [inline pkg/action/git.go]: This is super nitpicky at this point, but I'd remove the "new" here.

**Comment by campoy** [comment]
Happy to contribute to the tool, it's making my life much better (and my accounts safer!)

**Comment by muesli** [comment]
@campoy You should totally mention it in #justforfunc :stuck_out_tongue_winking_eye:

**Comment by dominikschulz** [comment]
Thanks a lot for contributing @campoy (and all your community work / justforfunc!)

We always try to improve this tool. Let us know if you miss anything.

@muesli thanks for helping out here :)

**Pull request merged**